src/Microsoft.ML.StandardLearners/Standard/MultiClass/OneVersusAllTrainer.cs

```diff
@@ -62,30 +62,27 @@ internal sealed class Options : OptionsBase
         }
 
         /// <summary>
-        /// Legacy constructor that builds the <see cref="Ova"/> trainer supplying the base trainer to use, for the classification task
-        /// through the <see cref="Options"/>.
-        /// Developers should instantiate OVA by supplying the trainer argument directly to the OVA constructor
-        /// using the other public constructor.
+        /// Constructs a <see cref="OneVersusAllTrainer"/> trainer supplying a <see cref="Options"/>.
         /// </summary>
         /// <param name="env">The private <see cref="IHostEnvironment"/> for this estimator.</param>
         /// <param name="options">The legacy <see cref="Options"/></param>
-        internal Ova(IHostEnvironment env, Options options)
+        internal OneVersusAllTrainer(IHostEnvironment env, Options options)
             : base(env, options, LoadNameValue)
         {
             _options = options;
         }
 
         /// <summary>
-        /// Initializes a new instance of <see cref="Ova"/>.
+        /// Initializes a new instance of <see cref="OneVersusAllTrainer"/>.
         /// </summary>
         /// <param name="env">The <see cref="IHostEnvironment"/> instance.</param>
         /// <param name="binaryEstimator">An instance of a binary <see cref="ITrainerEstimator{TTransformer, TPredictor}"/> used as the base trainer.</param>
-        /// <param name="calibrator">The calibrator. If a calibrator is not explicitely provided, it will default to <see cref="PlattCalibratorTrainer"/></param>
+        /// <param name="calibrator">The calibrator. If a calibrator is not provided, it will default to <see cref="PlattCalibratorTrainer"/></param>
         /// <param name="labelColumn">The name of the label colum.</param>
-        /// <param name="imputeMissingLabelsAsNegative">Whether to treat missing labels as having negative labels, instead of keeping them missing.</param>
+        /// <param name="imputeMissingLabelsAsNegative">If true will treat missing labels as negative labels.</param>
         /// <param name="maxCalibrationExamples">Number of instances to train the calibrator.</param>
```

Comment:
```suggestion
        /// <param name="maximumCalibrationExampleCount">Number of instances to train the calibrator.</param>
```
What's the indent scope of this PR? Inferred from its title, I assume that having better argument names is included. Please correct me if I am wrong.

Reply:
Better argument names is in scope for this PR.

---
In reply to: [261016830](https://github.com/dotnet/machinelearning/pull/2743#discussion_r261016830) [](ancestors = 261016830)